test/e2e/operator.go

```
@@ -329,6 +212,150 @@ func TestKCMRecovery(t *testing.T) {
+	err = testlib.WaitForServiceAccountInNamespace(kubeClient, name, "default")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	labels := map[string]string{"app": "pdbtest"}
+	err = pdbCreate(policyClient, name, minAvailable, maxUnavailable, labels)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = wait.PollImmediate(time.Second*1, testTimeout, func() (bool, error) {
```

golangci-lint (2.11.4)

[error] SA1019: wait.PollImmediate is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. 

(staticcheck)